CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## v10.4.1
```

Comment:
>4.1 [](start = 7, length = 3)

Shouldn't this be 10.5?

Reply:
Bleh you're right, I'll update.

Reply:
Fixed.  This also prompted me to notice that I forgot to update version.go 👍